proxies/live/apiproxy/resources/jsc/FilterScopesTokenExchange.js

```diff
@@ -1,8 +1,8 @@
 var api_product_scopes = String(context.getVariable('original_scope'));
 var idTokenIssuer = context.getVariable('idTokenIssuer');
 if (idTokenIssuer == "nhs-cis2") {
-    var id_token_acr = context.getVariable('jwt.DecodeJWT.FromSubjectTokenFormParam.decoded.claim.acr');
-    id_token_acr = ':' + id_token_acr.slice(0, 4).toLowerCase() + ':';
+    var id_token_acr = context.getVariable('jwt.DecodeJWT.FromSubjectTokenFormParam.decoded.claim.authentication_assurance_level');
+    //id_token_acr = id_token_acr;
 }
```

Comment:
As the authentication_assurance_level is just a number, I think it cause cause false positives if a number was included in the scope string without the cis2 context e.g. it would match `urn:nhsd:apim:app:level3:ambulance-analytics`. To improve the regex matching, perhaps we can have something like `id_token_acr = ':aal' + id_token_acr + ':'` . I think this is why the test is failing

Reply:
Still Test cases Failing even after making changes in the code.